service-map.json

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "Service",
+  "name": "officer-delta-processor",
+  "description": "The Officer Delta Processor is responsible for reading from the officer-delta kafka topic.",
```

Comment:
This is a little picky, but two points:
1. No need for the first bit - you could replace "The Officer Delta Processor is responsible for reading" with "Reads". We already know that this is the documentation for the Officer Delta Processor.
2. It reads from the topic and... what? What does it do with what is read from the topic?

Same points apply to the README.